### DISCUSSION TITLE
Error while using your package name "randomForestSRC"

### DISCUSSION THREAD

**Discussion by pijush1285** [kogalur/randomForestSRC#377] [General]
I was trying to use your package named “randomForestSRC” in survival analysis.
I found the error message provided below.

 > varImpPlot(rf_model)
Error in varImpPlot(rf_model) : could not find function "varImpPlot"

Would you kindly check it?
Thanks.

**Comment by ishwaran** [comment]
Hi, there is no such function in our package.

It seems like you are new to `randomForestSRC`, therefore I recommend visiting our [webpage](https://www.randomforestsrc.org).  In particular see [getting started](https://www.randomforestsrc.org/articles/getstarted.html) and [variable importance](https://www.randomforestsrc.org/articles/getstarted.html#variable-importance-vimp-and-dimension-reduction)

**Comment by pijush1285** [comment]
Hello,

Thank you for your reply.
Yes, I am very new to randomForestSRC.

Thanks,
Best regards,
Pijush
